apps/browser/settings/searchenginemodel.cpp

```diff
@@ -68,3 +69,40 @@ void SearchEngineModel::componentComplete()
 {
 
 }
+
+void SearchEngineModel::add(const QString& title, const QString& url)
+{
+    for (SearchEngine engine : m_searchEngines) {
```

Comment:
This should be like "const SearchEngine &engine : m_searchEngines" => now there will be copied.